src/SFA.DAS.AssessorService.Data/ScheduleRepository.cs

```diff
@@ -47,25 +50,33 @@ public async Task<ScheduleRun> GetNextScheduledRun(int scheduleType)
                                     FROM ScheduleRuns 
                                     WHERE ScheduleType = @scheduleType 
                                     AND IsComplete = 0 
-                                    ORDER BY RunTime", new {scheduleType});
+                                    AND (LastRunStatus <> @scheduleRunStatus or LastRunStatus is null)
```

Comment:
Should this filter out all except `Completed` runs?

Reply:
It is already filtering for Completed runs by IsComplete = 0, this is to stop getting the failed records